src/lib/coda_base/data_hash.ml

```diff
@@ -137,6 +137,28 @@ struct
   [%%endif]
 end
 
+module T0 = struct
+  module Arg = struct
+    type t = Field.t [@@deriving sexp, compare, hash]
+
+    [%%define_locally
+    Field.(to_string, of_string)]
+  end
+
+  [%%versioned_binable
+  module Stable = struct
+    module V1 = struct
+      type t = Field.t [@@deriving sexp, compare, hash]
+
+      type _unused = unit constraint t = Arg.t
+
+      let to_latest = Fn.id
+
+      include Binable.Of_stringable (Arg)
```

Comment:
Shouldn't this have serialisation tests?

Reply:
This was the issue #4644, should `%versioned_binable` modules require tests, which I decided against, in general.

This is a special case, though, where the argument to `Binable.Of_string` is imported, and the stringification could change. I'll add tests outside the `Stable` module.